scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
+
+However, the SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a
+minimum. On a normal ethernet network, this should be set to around 1450. On a
+jumbo frame ethernet network, this should be around 8950.
```

Comment:
I would remove the words "around" from this sentence.  1450 and 8950 are exactly what they should be.